### PR TITLE
Image overlays fixes

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes several things:

- Removes an unused loader.
- Centers the image overlay in the mobile layout.
- Fixes a minor issue with the line-height that created an extra padding in the overlay.
- In the editing mode the height of the overlay is no longer the height of the image, but the height of the URL.
- Fixes an issue that made a mobile overlay appear in the desktop view.

### CODE REVIEW

**Comment by javierarce** [comment]
@xavijam: can you have a look, please?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3436/))

File: lib/assets/javascripts/cartodb/table/overlays/image.js [hunk 1]
```diff
@@ -121,11 +121,15 @@ cdb.admin.overlays.Image = cdb.admin.overlays.Text.extend({
...
 
-    if (cdb.Utils.isURL(text)) {
-      self.model.set("url", text);
-    }
+      if (cdb.Utils.isURL(text)) {
+        self.model.set("url", text);
+        self.editModel.set("mode", "loading");
+      }
+
+    }, 300);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/overlays/image.js]: Why do you need this timeout?

File: lib/assets/javascripts/cartodb/table/overlays/image.js [hunk 2]
```diff
@@ -197,7 +201,6 @@ cdb.admin.overlays.Image = cdb.admin.overlays.Text.extend({
     this.$el
     .removeClass("editable")
     .removeClass("disabled")
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/overlays/image.js]:
```js
this.$el.removeClass(' editable disabled ')
```

Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/overlays/image.js]: Ok, thanks.

**Comment by xavijam** [comment]
Oooook!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3439/))

**Comment by javierarce** [comment]
Thaaaanks!

**Pull request merged**